test/node/symbol_tests.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const BSON = require('../register-bson');
+const util = require('util');
+const BSONSymbol = BSON.BSONSymbol;
+
+describe('MinKey tests', function () {
```

Comment:
```suggestion
describe('BSONSymbol tests', function () {
```

Reply:
oops, missed this one. Actually, now that we're here does it make sense to just add all of these tests in [bson_tests](https://github.com/mongodb/js-bson/blob/master/test/node/bson_test.js) instead? I'm imagining a future maintainer forgetting to change one of these in the future because they have to change ~10 files

Reply:
Yeah I wasn't sure how to handle that and then went with the individual files. I could implement a test case like https://github.com/mongodb/js-bson/blob/master/test/node/bson_test.js#L1310 which would `inspect` an object with all the different types and directly compare that instead of individual test cases?

Reply:
I like the idea of checking the behavior in one place, it looks like maybe there's a build failure related to formatting of the string. Maybe it's easier to do something like:

Reply:
Ah alright now I see - sometimes it has line breaks, sometimes it doesn't. Will fix asap and split it up into individual cases to circumvent that.

Reply:
I've now added individual cases per type in one block in `bson_tests` - let's hope these now work in evergreen 😄